processing/src/main/java/org/apache/druid/query/QueryToolChest.java

```diff
@@ -251,19 +251,30 @@ public Function<ResultType, ResultType> makePostComputeManipulatorFn(QueryType q
    */
   public abstract TypeReference<ResultType> getResultTypeReference();
 
+  /**
+   *
```

Comment:
nit: please add javadocs or remove (but preferably add and what does null mean if it returns null like the other one has).

Also, should this be deprecated?